internal/repo/repo_structure_exists.go

```diff
@@ -108,11 +107,21 @@ func checkMissingFiles(listOfFiles []string) []string {
 	return missingFiles
 }
 
-// Function to format output message containing missing files.
-func fmtMessage(startingMessage string, listOfFiles []string) string {
-	message := startingMessage
-	for _, missingFile := range listOfFiles {
-		message += missingFile + " "
+func CreateFiles(files []string) {
+	for _, file := range files {
+		os.Create(file)
+	}
+}
+
+func CreateDirs(dirs []string) {
+	for _, dir := range dirs {
+		os.Mkdir(dir, 0750)
```

Comment:
Be careful with `Mkdir`. It is only capable of creating one level of dirs. `MkdirAll` could be a better option